human_robot_collaboration/web-interface/speech_predicition/ros.js

```diff
@@ -191,8 +227,32 @@ function callback(e) {
               console.log('[left] Got Response: ' + res.success + ' ' + res.response);
           });
         }
-        else
+
+        // Begins rosbagging baxter data
+        else if (obj.includes("START"))
         {
+            var req = new ROSLIB.ServiceRequest();
+            var res = new ROSLIB.ServiceResponse();
+
+            console.log("Requested rosbag recording start!");
+            startExperiment.callService(req,function(res){
+                console.log("Got response!");
+            });
+        }
+
+        else if (obj.includes("STOP"))
+        {
+            var req = new ROSLIB.ServiceRequest();
+            var res = new ROSLIB.ServiceResponse();
```

Comment:
@houndci-bot has a point. We should define `req` and `res` once in the beginning, and reuse them later on.